packages/components/src/internal/components/user/actions.ts

```diff
@@ -158,26 +156,26 @@ export function getPasswordRuleInfo(): Promise<any> {
     });
 }
 
-export function getSelectedUserIds(model: QueryGridModel): List<number> {
+export function getSelectedUserIds(model: QueryModel): number[] {
     // selectedIds will be strings, need to cast to integers
-    return model.selectedIds.map(id => parseInt(id)).toList();
+    return Array.from(model.selections).map(id => parseInt(id));
```

Comment:
I wonder if this is a method that is generally useful.  I don't know of cases off the top of my head where we need this translation, but it drives me a little crazy that the rowIds are sometimes strings and sometimes integers.

Reply:
Sure. Move this to a helper method off of the QueryModel.